client/core/wallet.go

```diff
@@ -197,34 +200,51 @@ func (w *xcWallet) connected() bool {
 // flag to true, and validates the deposit address. Use Disconnect to cleanly
 // shutdown the wallet.
 func (w *xcWallet) Connect() error {
+	reloadWallet := func() {
+		wallet, err := asset.OpenWallet(w.AssetID, w.cfg, w.logger, w.net)
```

Comment:
a connect error now requires on an asset.Driver being registered for this asset ID, and these new xcWallet fields being set